utils.js

```diff
@@ -138,3 +142,23 @@ export const displayTimeEntry = async function (timeEntry) {
     console.info(`Workspace: ${workspace.name} (#${timeEntry.wid})`)
   }
 }
+
+/**
+ * Parses a timelike string into a dayjs object of the current date and that time
+ * @param {string} timeString timelike string e.g. 4:50PM '12:00 AM' etc.
+ * @returns {object} dayjs object
+ */
+export function parseTime (timeString) {
```

Comment:
Thank you for  moving this here, this really should have been available for reuse.